EssentialsDiscord/src/com/earth2me/essentials/discord/EssentialsJDA.java

```diff
@@ -54,8 +73,28 @@ public void sendMessage(long channelId, String message) {
             logger.log(Level.WARNING, tl("discordInvalidChannelId"));
             return;
         }
-        String messageStripped = ChatColor.stripColor(message);
-        channel.sendMessage(messageStripped).queue(success -> {
+        String msg = ChatColor.stripColor(message)
+                .replaceAll("@here", "")
+                .replaceAll("@everyone", "");
+        if (msg.contains("#")) {
+            for (TextChannel textChannel : guild.getTextChannels()) {
+                msg = msg.replaceAll("#" + textChannel.getName(), textChannel.getAsMention());
+            }
+        }
+        if (Pattern.compile(":.+:").matcher(msg).find()) {
+            for (Emote emote : guild.getEmotes()) {
+                msg = msg.replaceAll(":" + emote.getName() + ":", emote.getAsMention());
+            }
+        }
+        if (msg.contains("@")) {
+            for (Role role : guild.getRoles()) {
+                msg = msg.replaceAll("@" + role.getName(), role.getAsMention());
+            }
+            for (Member member : guild.getMembers()) {
+                msg = msg.replaceAll("@" + member.getEffectiveName(), member.getAsMention());
+            }
+        }
+        channel.sendMessage(msg).queue(success -> {
```

Comment:
this should allow no mentions in the message 
```suggestion
        channel.sendMessage(msg).allowedMentions(new ArrayList<>()).queue(success -> {
```